speakerfight/settings.py

```diff
@@ -40,6 +40,7 @@
 
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
 DEFAULT_FROM_EMAIL = NO_REPLY_EMAIL = 'Speakerfight <noreply@speakerfight.com>'
+FEEDBACK_EMAIL = 'Speakerfight <feedback@speakerfight.com>'
```

Comment:
Makes sense, @kerollaine, but if this e-mail address will only notify the webmaster (do we use this term yet? haha) about users’ feedback then there’s no need to receive responses and we can stuck with noreply@[...].